docs/regmap/adi_regmap_spi_engine.txt

```diff
@@ -447,6 +447,21 @@ ENDFIELD
 ############################################################################################
 ############################################################################################
 
+REG
+0x43
+OFFLOAD0_SDO_SRC_SEL
```

Comment:
I already submitted patches to upstream Linux assuming that this was a compile time option instead of a runtime option. So if we are going to change this, I need to retract the changes in that patch.

However, having the setting globally like this instead of per transfer isn't as useful since it would limit a SPI offload to either using one or the other. We could never have a SPI message that does both types of TX transfers. So I would be OK if we just left this as a compile time option for now.